### ISSUE TITLE
adhere to ARIA listbox standards

### ISSUE THREAD

**Issue by trippel** [jackocnr/intl-tel-input#1145]
Hi,

According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role every selectable, but not selected listbox child of the type option should have aria-selected="false" set.

Currently only the childs which have been selected before have aria-selected="false" set.

I think that it is a good idea to adhere to this reference and set all not selected, but selectable listbox childs to aria-selected="false".

Best regards

**Comment by jackocnr** [comment]
Thanks for reporting this, we actually have an open PR, I just need to get around to reviewing it 😅  https://github.com/jackocnr/intl-tel-input/pull/1144

**Comment by trippel** [comment]
Thanks for your prompt answer. Then, I will happily wait until you find the time to merge the PR. :)
